### ISSUE TITLE
Using google oAuth on localhost windows environemtn

### ISSUE THREAD

**Issue by rudolfbruder** [nuxt-community/auth-module#1354]
Hi,

I managed to install this package on my project. However I am unable to test it out locally because my nuxt project starts on http://localhost:3000 and google oAuth needs HTTPS address.

I really did not find any way how to perform this locally on windows environment. I managed to do this on my VPS with nginx and reverse proxy etc, but how can I test this out and actually use it locally?

My problem is that I cannot specify the redirect uri to localhost:3000 because google won't accept it and I think what is needed is to actually have https with custom domain name running locally.

Could someone please help me out here?

Thank you so much!

**Comment by Intevel** [comment]
I don't think there is really a solution, Google only accepts SSL/TLS and they are not the only ones. This is not a nuxtauth question

**Comment by bmulholland** [comment]
I agree that this isn't a nuxt auth question, but I did set this up for my local development using the devcert npm module. Here's my config -- sorry, I don't have bandwidth to provide support on this issue beyond posting it:

```
export default async () => {
  let server

  if (process.env.HTTPS) {
    // eslint-disable-next-line no-console
    console.log(
      "Installing and trusting local certs; you may get prompts to enter your password"
    )
    // cert lib takes care of creating and trusting the local HTTPS cert
    const certs = await devcert.certificateFor("localhost")

    server = {
      https: {
        key: certs.key.toString(),
        cert: certs.cert.toString(),
        port: 3000,
      },
    }
  } else {
    server = {}
  }

  return {
    server,
```
